spec/features/admin_matching_attributes_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.feature "Admin checking a claims" do
```

Comment:
Can this be reworded to better describe the problem we are trying to solve with this feature? e.g. `"Service operator can see potential duplicate claims"`. Same goes for the filename. As a new developer to the project, if I read `admin_matching_attributes_spec.eb`, what does that say to me about the feature? Could the file be named to better express the intent of the feature?

Reply:
Agreed - I'll change it to something more descriptive.